tests/test_api.py

```diff
@@ -14,19 +14,148 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
```

Comment:
Please tiny test files that are just a few bytes. https://github.com/nexB/fetchcode/blob/4ad6ca4384e789e4b3cff5e1daf28a927f07be7f/tests/data/img.png is  way too big we like to try to avoid polluting the version control history with big test files when possible.

Furthermore it may a copyrighted image?

Reply:
Okay got this one